### DISCUSSION TITLE
複数のタブで同じ記事を編集した時の上書き防止が欲しい（楽観的ロックの実装）

### DISCUSSION THREAD

**Discussion by ko1nksm** [increments/qiita-discussions#198] [要望・アイデア・不具合]
複数のタブやマシンで同じ記事を気づかずに

1. Aタブで編集開始（保存ボタンは押さない）
2. Bタブで開始開始（保存ボタンは押さない）
3. Aタブで編集（Aタブの内容で保存される）
4. Bタブで編集（Bタブの内容で保存される）

してしまうとAタブでの編集が失われてしまいます。
いわゆる楽観的ロックが実装されていると嬉しいかなぁと思います。

**Comment by kskwtnk** [comment]
@ko1nksm 
ご意見ありがとうございます！

こちら……正直に申し上げますと「共感できるものの、だいぶ難しそう」という状態です 🙇 
実現したいのは間違いないのですが、直近の開発優先度などから考えると先になってしまうかもしれません……

ただ、実際にご意見がいただけたものとなると社内での議論も活発になりますので今後も是非Discussionにてご要望をいただけますと幸いです 🙇
